test/integration/smoke/test_list_ids_parameter.py

```diff
@@ -0,0 +1,106 @@
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" Tests for API listing methods using 'ids' parameter
+"""
+#Import Local Modules
+from marvin.cloudstackTestCase import cloudstackTestCase
+from marvin.lib.utils import (cleanup_resources,
+                              validateList)
+from marvin.lib.base import (Account,
+                             Volume,
+                             DiskOffering)
+from marvin.lib.common import (get_domain,
+                                get_zone)
+from marvin.codes import FAILED, PASS
+from nose.plugins.attrib import attr
+#Import System modules
+import time
+
+_multiprocess_shared_ = True
+class TestListIdsParams(cloudstackTestCase):
+    
+    @classmethod
+    def setUpClass(cls):
+        testClient = super(TestListIdsParams, cls).getClsTestClient()
+        cls.apiclient = testClient.getApiClient()
+        cls.services = testClient.getParsedTestDataConfig()
+        cls.hypervisor = testClient.getHypervisorInfo()
+
+        cls.domain = get_domain(cls.apiclient)
+        cls.zone = get_zone(cls.apiclient, testClient.getZoneForTests())
+
+        cls.disk_offering = DiskOffering.create(
+                                    cls.apiclient,
+                                    cls.services["disk_offering"]
+                                    )
+
+        cls.account = Account.create(
+                            cls.apiclient,
+                            cls.services["account"],
+                            domainid=cls.domain.id
+                            )
+
+        cls.volumes = []
+        testVolumes = {0: {'diskname': 'TestDiskServ'}, 1: {'diskname': 'TestDiskServ2'}, 2: {'diskname': 'TestDiskServ3'}}
+
+        for k, v in testVolumes.items():
+            volume = Volume.create(
+                                   cls.apiclient,
+                                   v,
+                                   zoneid=cls.zone.id,
+                                   account=cls.account.name,
+                                   domainid=cls.account.domainid,
+                                   diskofferingid=cls.disk_offering.id
+                                   )
+            cls.debug("Created a volume with ID: %s" % volume.id)
+            cls.volumes.append(volume)
+
+        cls._cleanup = [
+                        cls.disk_offering,
+                        cls.account
+                        #,cls.volumes[0],
```

Comment:
@nvazquez , 
Sorry to let you down. I am terrible with python.
I believe @rhtyd,  @swill,  or @DaanHoogland can help you with that.